llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -269,7 +269,7 @@ class TargetInstrInfo : public MCInstrInfo {
   /// the destination along with the FrameIndex of the loaded stack slot.  If
   /// not, return 0.  This predicate must return 0 if the instruction has
```

Comment:
Nit: update this (and other) comment to say `NoRegister` instead of `0`? Also maybe "return the ... register" instead of "return the ... register number"?